brokerapi/brokers/cloudsql/postgres-definition.go

```diff
@@ -128,10 +134,68 @@ func init() {
 				    }
 				]
     }`,
-		ProvisionInputVariables: commonProvisionVariables,
-		DefaultRoleWhitelist:    roleWhitelist,
-		BindInputVariables:      commonBindVariables,
-		BindOutputVariables:     commonBindOutputVariables,
+		ProvisionInputVariables: append([]broker.BrokerVariable{
+			{
+				FieldName: "instance_name",
+				Type:      broker.JsonTypeString,
+				Details:   "Name of the CloudSQL instance.",
+				Default:   "pcf-sb-${counter.next()}-${time.nano()}",
+				Constraints: validation.NewConstraintBuilder().
+					Pattern("^[a-z][a-z0-9-]+$").
+					MaxLength(75).
+					Build(),
+			},
+			{
+				FieldName: "database_name",
+				Type:      broker.JsonTypeString,
+				Details:   "Name of the database inside of the instance. Must be a valid identifier for your chosen database type.",
+				Default:   "pcf-sb-${counter.next()}-${time.nano()}",
+			},
+			{
+				FieldName: "version",
+				Type:      broker.JsonTypeString,
+				Details:   "The database engine type and version.",
+				Default:   "POSTGRES_9_6",
+				Enum: map[interface{}]string{
+					"POSTGRES_9_6": "PostgreSQL 9.6.X",
+				},
+			},
+			{
+				FieldName: "failover_replica_name",
```

Comment:
Based on the docs I'm not convinced that this is the correct way to create a HA Postgres instance.